clients/database/postgres/partition.go

```diff
@@ -0,0 +1,260 @@
+package postgres
+
+import (
+	"context"
+	"fmt"
+	"github.com/kava-labs/kava-proxy-service/config"
+	"math"
+	"strings"
+	"time"
+)
+
+const (
```

Comment:
Migrated partition part of the code into database, as it is based on the implementation details of the db itself